tests/acceptance/features/webUIUpload/uploadEdgecases.feature

```diff
@@ -51,14 +51,16 @@ Feature: File Upload
 
 
   Scenario: overwrite an existing file
+    Given user "Alice" has created file "'single'quotes.txt" in the server
+    And user "Alice" has created file "strängé filename (duplicate #2 &).txt" in the server
+    And user "Alice" has created file "zzzz-must-be-last-file-in-folder.txt" in the server
```

Comment:
how did this work before?

Reply:
previously `confirmFileOverwrite` function did fail but the test kept on going.